### ISSUE TITLE
Setting environment variables when activating conda env

### ISSUE THREAD

**Issue by nhz2** [cjdoris/CondaPkg.jl#92]
Conda packages can have scripts to set and unset environment variables when the environment is activated and deactivated.

https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#saving-environment-variables

These get added to `.CondaPkg/env/etc/conda/activate.d/` and `.CondaPkg/env/etc/conda/deactivate.d/`

Currently from what I can tell `CondaPkg.withenv(f::Function)` doesn't run these scripts.

**Comment by cjdoris** [comment]
Indeed, most Conda packages do not use this feature, so it hasn't been implemented by CondaPkg.

Are there particular packages you're having issue with.

**Comment by cjdoris** [comment]
Indeed, an earlier version of CondaPkg did this, but it was not super reliable. It could be reimplemented though.
